.gitignore

```diff
@@ -7,3 +7,4 @@ node_modules
 .env
 /styleguide
 .coverage
+.DS_Store
```

Comment:
I can still see `src/.DS_Store`, `src/components/.DS_Store` and `src/components/Dashboard/.DS_Store` commited:

![screen shot 2018-09-27 at 8 22 55 am](https://user-images.githubusercontent.com/3766511/46145617-b2de5700-c22e-11e8-85aa-1b4b48df7ffd.png)

You'll have to do something like:

```bash
git rm src/.DS_Store
git rm src/components/.DS_Store
git rm src/components/Dashboard/.DS_Store
git commit -m "remove DS_Store"
```

Then the last step would be to `git push` to this branch.

Reply:
Thanks for the help and I have just done finished the push. =)